### PR TITLE
1907-12-09 pg2-3

### PR DESCRIPTION
**Pull request by bmh19**
The second commit, which should be all green is the correct one :)

### CODE REVIEW

**Comment by huoyanzhizhou** [comment]
Several little things: Page 3 is not its page div; please double check when you update the information from the template. For example, line 15 still has the old date information; Local and General can be a section rather than an item with a feature="local".

**Comment by whanley** [comment]
Hi Breanna. Deliverables 1,2, and 3 need to be in a single file, named 1907-12-09.xml, which is the entire Monday issue. Tuesday will be a second file, named 1907-12-10.xml. This pull request should contain no other files. Your XPath query is not part of the content repository; you'll submit it differently. Please put all of your work in one file, delete the rest of the files, and send a new pull request.

**Comment by whanley** [comment]
Thanks! Also remove 1907-12-09 D3.xml,  Boilerplates.xml, and Xpath Query & Discussion.docx

**Comment by bmh19** [comment]
Did you get the file 1907-12-09? I'm not sure if the pull request went through...

Breanna
________________________________

**Comment by whanley** [comment]
Yes, this is just right now. Good. I see that you still have material to fill in for this issue, but generally: good work! Send a new pull request when you have more OCR fixed up and templates completed.

**Comment by bmh19** [comment]
Thank you!!! So each time I send I send a new pull request, do I delete the old version?
________________________________

**Comment by whanley** [comment]
No, you don't need to delete the old version. After you commit changes and push your file, you can click the pull request button and your new work will be merged into this pull request.

**Comment by bmh19** [comment]
Ok, thank you
________________________________

**Comment by bmh19** [comment]
1907-12-11 and 1907-12-12 are not complete yet, put I am in the process of finishing them!

**Comment by whanley** [comment]
Hi Breanna. You are doing lots of good work--but there's obviously more to do.
- templates and ads need to be updated to reflect contents of _your_ issues
- you are having some odd results out of your OCR, particularly with respect to too many paragraph breaks, and sections which you report do not OCR. Have you tried other methods? Come and talk to me or Weber if you want to figure out how to save formatting work as you import your OCR into XML (or check the tutorials again).
- on your Monday issue, at least, there are some divs that are inside other divs, but should be on their own (particularly on page 3). Make sure you keep siblings on the same level.
Thanks for your efforts--I look forward to seeing the final results!

**Comment by bmh19** [comment]
Ok, I can go back and update templates and fix the errors that you refer to, thank you for letting me know though. When I place my OCR into XML in the author mode it automatically puts all those paragraph breaks and I wasn't sure if that mattered, so yes help would that would be nice. I was planning on going back to old issues and trying the "Google Drive" Method, seems to work in some of the areas that OCR doesn't.

Thank you,
Breanna Heflin


________________________________

**Comment by whanley** [comment]
Sounds good. Come to office hours or make an appointment and we can help you with it.

File: 1907-12-11.xml [hunk 1]
```diff
@@ -0,0 +1,7587 @@
...
+                            <cell>Shellal</cell>
+                            <cell>depart</cell>
+                            <cell>Sundays and Wedn.</cell>
+                            <cell>9.30 p.m.</cell>
+                        </row>
+                        <row>
+                            <cell>Wednesdays and Saturdays‡</cell>
+                            <cell>5.30 p.m.</cell>
+                            <cell>depart</cell>
+                            <cel
```
Comment (huoyanzhizhou) [inline 1907-12-11.xml]: Please make small essays in Local and General into individual divs.

Comment (huoyanzhizhou) [inline 1907-12-11.xml]: Remember to update your measures

**Comment by huoyanzhizhou** [comment]
Some good updates on 12-09, please keep your good work on other files.

**Comment by whanley** [comment]
Let's take a look at this pull request next week, Breanna.

**Comment by whanley** [comment]
Good stuff. But we need to spend a bit of time tidying things up in one of the work sessions in Strozier.

**Pull request merged**